Baikal/RenderFactory/render_factory.h

```diff
@@ -35,6 +35,7 @@ namespace Baikal
 
     enum class PostEffectType
     {
+        kNone,
```

Comment:
Why we need kNone here ?

Reply:
It is to distinguish cases where the `denoiser_type` parameter is not specified.

Reply:
DenoiserMode::kNone is enough